pwndbg/commands/vmmap.py

```diff
@@ -44,6 +49,100 @@ def print_vmmap_table_header() -> None:
     )
 
 
+def print_vmmap_gaps_table_header() -> None:
+    """
+    Prints the table header for the vmmap --gaps command.
+    """
+    header = [f"{'Start':>{2 + 2 * pwndbg.gdblib.arch.ptrsize}}"]
+    header.append(f"{'End':>{2 + 2 * pwndbg.gdblib.arch.ptrsize}}")
+    header.append(f"{'Perm':>4}")
+    header.append(f"{'Size':>8}")
+    header.append(f"{'Note':>9}")
+    header.append(f"{'Accumulated Size':>{2 + 2 * pwndbg.gdblib.arch.ptrsize}}")
+    print(" ".join(header))
```

Comment:
You can construct a multi-line f-string like this:
```py
In [1]: x=1

In [2]: y=2

In [3]: s = (
   ...:     f'{x}\n'
   ...:     f'{y}\n'
   ...:     f'That works'
   ...: )

In [4]: print(s)
1
2
That works
```

Let's do that here; we don't really need to construct list/append/join :)